content/documentation/staging/installation-guide/index.adoc

```diff
@@ -288,6 +288,8 @@ The Kiali CR tells the Kiali Operator which namespaces are accessible to Kiali.
 
 The specified namespaces are those that have service mesh components to be observed by Kiali. Additionally, the namespace to which Kiali is installed must be accessible (typically the same namespace as Istio). Each list entry can be a regex matched against all namespaces the operator can see. If not set in the Kiali CR, then the default behavior makes all current namespaces accessible except for some internal namespaces that should typically be ignored.
 
+icon:bullhorn[size=1x]{nbsp} If you install multiple Kiali instances in the cluster, they must be configured with completely independent sets of accessible namespaces. In other words, a namespace must only ever appear in the `accessible_namespaces` list for at most one Kiali instance.
```

Comment:
suggested change:  

When installing multiple Kiali instances into a single cluster, the accessible namespaces must be mutually exclusive. In other words, a namespace must appear in one and only one `accessible_namespaces` list.  Regular expressions must not have overlapping patterns.

Reply:
Instead of `must appear in one and only one` I'm going to say `must appear in at most one`. Because by saying "one and only one" it implies all namespaces MUST be in at least one accessible_namespace list (`must appear in one...`) which is not true. A namespace need not be known to any Kiali instance.